### ISSUE TITLE
Fix the footer for landscape Mode!!

### ISSUE THREAD

**Issue by DARK-art108** [girlscript/EQEquation-Website#104]
![Screenshot_20201012-123546__01](https://user-images.githubusercontent.com/53623244/95716151-9cc25880-0c88-11eb-9382-f8026cded8fe.jpg)

**Comment by mahimagoyalx** [comment]
I would like to take this issue, please elaborate on what needs to be fixed. @DARK-art108

**Comment by ishi0301** [comment]
I would like to help @mahimagoyalx in this

**Comment by saviourcode** [comment]
@mahimagoyalx, As you can see the footer is shifted to the left and not at the center.
Can you make the footer centered with respect to the page like shown below?
![footersample](https://user-images.githubusercontent.com/36518515/95719684-ce89ee00-0c8d-11eb-9b33-f61c086e226f.PNG)

**Comment by saviourcode** [comment]
@ishi0301 , It is not that big a task and can be handled by an individual as well. You can take other issues that are currently open.

**Comment by mahimagoyalx** [comment]
> @mahimagoyalx, As you can see the footer is shifted to the left and not at the center.
> Can you make the footer centered with respect to the page like shown below?
> ![footersample](https://user-images.githubusercontent.com/36518515/95719684-ce89ee00-0c8d-11eb-9b33-f61c086e226f.PNG)

Yeah, I can do that, can you please tell me, where exactly the problem is occurring, because I find it fine here. [
<img width="959" alt="Capture" src="https://user-images.githubusercontent.com/44669552/95720396-d1d1a980-0c8e-11eb-9bb8-e962a306a5c6.PNG">
](url)

**Comment by mahimagoyalx** [comment]
@saviourcode @DARK-art108 I got it, please assign me this issue.

**Comment by saviourcode** [comment]
@mahimagoyalx All the best!

**Comment by mahimagoyalx** [comment]
For less than 1200px width
<img width="788" alt="lessthan1200" src="https://user-images.githubusercontent.com/44669552/95844885-16307880-0d67-11eb-90da-e4b8b5330413.PNG">

For more than 1200px width
<img width="960" alt="morethan1200" src="https://user-images.githubusercontent.com/44669552/95844899-1b8dc300-0d67-11eb-9df8-dc02c7afeda7.PNG">

Should I make the pull request? @saviourcode @DARK-art108

**Comment by saviourcode** [comment]
@mahimagoyalx is it possible to create just a one singe row of the all the column with some space between them? 
Like there is still some space in between the left side of the footer and logo

**Comment by saviourcode** [comment]
> ![footersample](https://user-images.githubusercontent.com/36518515/95719684-ce89ee00-0c8d-11eb-9b33-f61c086e226f.PNG)
> 

Reference this picture for final result

**Comment by mahimagoyalx** [comment]
Then for less than 1200px width all the columns will compress, come really close to each other and won't look good.

**Comment by saviourcode** [comment]
Yeah, that's why we need to increase the spacing between them. If you want you might consider reducing logo size for more space.

**Comment by saviourcode** [comment]
@mahimagoyalx Is the issue complete :)

**Comment by himanshubobade** [comment]
hey i would like to work on it

**Comment by saviourcode** [comment]
@himanshubobade sorry but this issue is fixed. You may check other open issues

**Comment by himanshubobade** [comment]
@saviourcode okay i will work on that
